packages/discovery-styles/scss/global-carbon-styles.scss

```diff
@@ -0,0 +1,14 @@
+// You only need to import this file if your app is not already directly importing these carbon files
+
+$css--plex: true !default;
+$css--use-layer: true !default;
+
+// Global styles
+@import '~carbon-components/scss/globals/scss/_css--font-face.scss';
+@import '~carbon-components/scss/globals/scss/_typography.scss';
+@import '~carbon-components/scss/globals/scss/_colors.scss';
+@import '~carbon-components/scss/globals/scss/_layout.scss';
+
+// Search Input
+@import '~carbon-components/scss/components/search/search';
+@import '~carbon-components/scss/components/list-box/list-box';
```

Comment:
This file is completely optional for consumer apps. They can choose to import this file along with the base discovery components scss file if they don't plan on mixing in any additional carbon components. I imagine this would be more likely used by users who are quickly building a sample search app. Perhaps as their project grows, they would abandon importing this file in favor of importing from carbon directly. Bottom line is that this allows to provide a decent developer experience for those who want a quick path to get something going while also saying "hey, you don't need to use this file if you have a more complex setup to manage."